renovate.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": ["config:js-lib"],
+  "enabledManagers": ["npm"],
+  "updateInternalDeps": true,
+  "packageRules": [
+    {
+      "packagePatterns": ["^@bbc/psammead"],
```

Comment:
Are we going to start with this rule initially or exclude psammead packages and let talos continue to do its thing?

Reply:
Keeping Talos 👍